### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by acinader**
the current winston and parse versions are incompatible.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -27,9 +27,9 @@
   },
   "homepage": "https://github.com/montymxb/parse-server-test#readme",
   "dependencies": {
-    "forever": "0.15.3",
+    "forever": "1.0.0",
     "mongodb-runner": "4.7.1",
-    "parse-server": "github:parse-community/parse-server#master",
-    "winston": "2.4.3"
+    "parse-server": "3.3.0",
```
Comment (dplewis) [inline package.json]: Can you revert this to point to the latest branch?

**Review by dplewis — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (acinader) [reply package.json]: why?  then if the master branch is broken, the php repo will be broken too? seems better to test against the current release?

Comment (dplewis) [reply package.json]: If a new feature is added we would have to wait for a server release (which would take a while) before we can update the SDK's. It's the same for the JS SDK.

If master was broken we would know sooner.

Comment (acinader) [reply package.json]: k.  done.

**Review by dplewis — APPROVED** [review]

**Comment by dplewis** [comment]
Thanks!

**Pull request merged**